erpnext/accounts/doctype/journal_entry/journal_entry.js

```diff
@@ -451,10 +451,20 @@ frappe.ui.form.on("Journal Entry Account", {
 	},
 
 	debit_in_account_currency: function(frm, cdt, cdn) {
+		var row = locals[cdt][cdn];
+		if(row.debit_in_account_currency < 0 || row.debit_in_account_currency === 0){
+			frappe.model.set_value(cdt, cdn, "debit_in_account_currency", '');
+			frappe.throw(__("Debit Amount cannot be zero or negative."));
```

Comment:
Debit / credit amount can be zero

Reply:
Then what is the use of creating JV with zero amount.?

Reply:
In a specific row, either debit or credit will have value, other will be zero (or blank).